server/blob/blob-postgres/src/main/java/org/apache/james/blob/postgres/PostgresBlobStoreDAO.java

```diff
@@ -48,11 +48,11 @@
 import reactor.core.publisher.Mono;
 
 public class PostgresBlobStoreDAO implements BlobStoreDAO {
-    private final PostgresExecutor postgresExecutor;
+    private final PoolPostgresExecutor postgresExecutor;
```

Comment:
Is there other DAOs that should benefit from the pool? (could be done in other task though if yes)

Reply:
Rely on the PgExecutor interface. Don;t mandate the pool.

Reply:
I created an interface, with 2 implements for it